### DISCUSSION TITLE
zh/tools/chartrace/dynamic/income-growth

### DISCUSSION THREAD

**Discussion by giscus** [selfboot/ai_gallery#6] [Q&A]
# zh/tools/chartrace/dynamic/income-growth

全球收入的变化经历了巨大的不平衡发展，早期由工业革命推动的发达国家收入增长占主导，后来新兴经济体的崛起带来了收入格局的转变，但全球收入差距依然显著。不同地区的收入增长率差异明显。

https://gallery.selfboot.cn/zh/tools/chartrace/dynamic/income-growth

**Comment by RealPeopleOfChina** [comment]
这个好。

**Comment by RealPeopleOfChina** [comment]
______________________这是来自QQ邮箱的假期自动回复邮件。您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
